tests/parsing/GlobbingTest.py

```diff
@@ -195,8 +195,8 @@ def test_questionmark(self):
 
     def test_asterisk(self):
         pattern = 'a*b'
-        matches = ['axb', 'ayb']
-        non_matches = ['aXbX', os.path.join('a', 'b')]
+        matches = ['axb', 'ayb', os.path.join('a', 'b')]
+        non_matches = ['aXbX']
```

Comment:
Then how will pattern `a**b` and `a*b` ?

Reply:
I don't understand what you mean by that question